src/components/add-update-EventPage/AddEventPage.jsx

```diff
@@ -1,196 +1,29 @@
 import React, { useState } from 'react'
 import './AddEventPage.css'
 import {postEvent} from './AddEvent.js'
+import Form from './Form.jsx'
 
 
 
 
-export default function AddEventPage() {
 
+export default function AddEventPage() {
 
-    const [event_title,setevent_title]=useState("");
-    const [imgUrl,setimgUrl]=useState("");
-    const [maxParticipants,setmaxParticipants]=useState(0);
-    const [location,setlocation]=useState("");
-   const [formsummary, setFormsummary] = useState({
+  const event ={
+    event_title :'',
+    imgUrl : '',
+    maxParticipants : '',
+    location : '',
     summary: '',
     description: '',
-  });
-   const [formDate, setFormDate] = useState({
     date: '',
     event_start_time: '',
-  });
-  const [items, setItems] = useState([]);
-  const [newItem, setNewItem] = useState("");
-
-
-  
-
-
-
-  function handleSubmit(e){
-    e.preventDefault();
-    const newEvent = {
-    event_title : event_title,
-    imgUrl: imgUrl,
-    maxParticipants :maxParticipants ,
-    location:location,
-    formsummary :formsummary ,
-    formDate :formDate,
-    items :items
-    };
-    postEvent(newEvent)
-    .then(() => {
-      setevent_title("");
-      setimgUrl("");
-      setmaxParticipants(0);
-      setlocation("");
-      setFormsummary({ summary: "", description: "" });
-      setFormDate({ date: "", event_start_time: "" });
-     //setItems([]);
-  
-    })
-    .catch((error) => {
-        console.error("Failed to submit event:", error);
-    });
-   
-
+    items: [] ,
+    newItem : ''
```

Comment:
Please move the newItem property out of the event object and handle it separately within the component's state, as it is only used to track user input for adding items to the items array.